opengrok-indexer/src/main/java/org/opengrok/indexer/web/messages/MessagesContainer.java

```diff
@@ -121,7 +127,9 @@ public void addMessage(final Message m) {
             }
 
             if (isMessageLimitExceeded()) {
-                return;
+                LOGGER.log(Level.WARNING, "cannot add message to the system, limit of {0} messages exceeded." +
+                        "consider setting the ''messageLimit'' tunable.", messageLimit);
```

Comment:
Odd spacing.

Also maybe it might be more terse as `"exceeded Configuration messageLimit of {0}"`?

Reply:
yes, that reads better.